### PR TITLE
Gnome 3.28 compatibility fixes for #182

### PR DESCRIPTION
**Pull request by goodwillcoding**
add global.display.get_monitor_neighbor_index that uses global.screen.get_monitor_neighbor_index
Meta.DisplayDirection = Meta.ScreenDirection;

### CODE REVIEW

File: kludges.js [hunk 1]
```diff
@@ -44,6 +44,16 @@ if (!global.display.get_monitor_scale) {
...
 
+if (!global.display.get_monitor_neighbor_index) {
+    // `get_monitor_neighbor_index` polyfill a fallback for 3.28
+    global.display.constructor.prototype.get_monitor_neighbor_index = function(...args) {
+        return global.screen.get_monitor_neighbor_index(...args);
+    }
+}
+
+// polyfill for 3.28
+Meta.DisplayDirection = Meta.ScreenDirection;
```
Comment (olejorgenb) [inline kludges.js]: Thanks :) 

This line need to be in a guard(?) eg.:

```
if (!Meta.DisplayDirection && Meta.ScreenDirection) {
    Meta.DisplayDirection = Meta.ScreenDirection;
}
```

Comment (goodwillcoding) [reply kludges.js]: yup, I got trigger happy. fix pushed.

Comment (goodwillcoding) [reply kludges.js]: thanks for reviewing, will pay more attention in the future

**Comment by hedning** [comment]
Cheers :) Will try to remember the screen/display change next time I touch similar code.

**Pull request merged**